py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -392,6 +392,25 @@ def aten_ops_sigmoid(
     )
 
 
+@dynamo_tensorrt_converter(torch.ops.aten.sym_size.int)
+def aten_ops_symsize_int(
+    ctx: ConversionContext,
+    target: Target,
+    args: Tuple[Argument, ...],
+    kwargs: Dict[str, Argument],
+    name: str,
+) -> Union[TRTTensor, Sequence[TRTTensor]]:
+    return impl.shape.shape(
+        ctx,
+        target,
+        SourceIR.ATEN,
+        name,
+        args[0],
+        args_bounds_check(args, 1, None),
```

Comment:
This should be `args[1]`, since the schema, [here](https://github.com/pytorch/pytorch/blob/0978482afa69118da1a986a4edec3acea01d2c6d/aten/src/ATen/native/native_functions.yaml#L5324), requires that this argument be specified and an integer.